submissions/serializers.py

```diff
@@ -90,6 +90,14 @@ class ScoreSerializer(serializers.ModelSerializer):
     # Ensure that the created_at datetime is not converted to a string.
     created_at = DateTimeField(format=None, required=False)
 
+    annotation_types = serializers.SerializerMethodField()
+    def get_annotation_types(self, obj):
+        """
+        Inspect ScoreAnnotations to attach the annotation_type of any found annotations.
+        """
+        annotations = ScoreAnnotation.objects.filter(score_id=obj.id)
+        return [annotation.annotation_type for annotation in annotations]
```

Comment:
A `ScoreAnnotation` consists of an `annotation_type`, a `reason`, and a `creator`, all linked to a specific `Score` via a FK relationship ([source](https://github.com/edx/edx-submissions/blob/master/submissions/models.py#L323-L335)). I didn't think the reason or creator needed to be exposed at this level, so just went with the `annotation_type`.

Do you think we'd be better off exposing all that information in a dict like this?
```
'annotations':
  [{
    'type': 'staff_defined',
    'reason': 'B/c I said so',
    'creator': 12345,
  }]
```